scripts/generator/graphql_generator/csharp/Editor/ShopifyIosPostProcessor.cs.erb

```diff
@@ -0,0 +1,43 @@
+namespace <%= namespace %>.SDK {
+    using UnityEngine;
+    using UnityEditor;
+    using UnityEditor.iOS.Xcode;
+    using System.IO;
+    using System;
+
+    public class ShopifyIosPostProcessor {
+        public static void Process(string buildPath) {
+            string testPath = Path.Combine(buildPath, "Libraries/Plugins/iOS/Shopify/BuyTests/");
+            var testDirectory = new DirectoryInfo(testPath);
+            var project = new ExtendedPBXProject(buildPath);
+
+            try {
+                project.SetFilesInDirectoryToTestTarget(testDirectory);
+            } catch (Exception e) {
+                Debug.Log(e.Message);
+            }
+
+            // Removes default Unity_iPhone_Tests from the Test Target
+            project.RemoveFileFromBuild(project.TestTargetGuid, project.FindFileGuidByProjectPath("Unity-iPhone Tests/Unity_iPhone_Tests.m"));
+            project.SetBuildProperty(project.GetAllTargetGuids(), ExtendedPBXProject.SwiftVersionKey, "3.0");
+            project.SetBuildProperty(project.UnityTargetGuid, ExtendedPBXProject.RunpathSearchKey, "@executable_path/Frameworks");
+            project.SetBuildProperty(project.TestTargetGuid, ExtendedPBXProject.RunpathSearchKey, "@loader_path/Frameworks");
+            project.SetBuildProperty(project.TestTargetGuid, ExtendedPBXProject.ProjectModuleNameKey, "$(PRODUCT_NAME:c99extidentifier)Tests");
+            project.SetBuildPropertyForConfig(project.GetDebugConfig(project.UnityTargetGuid), ExtendedPBXProject.EnableTestabilityKey, "YES");
+
+            #if UNITY_5_5_OR_NEWER
+            if (float.Parse(UnityEditor.PlayerSettings.iOS.targetOSVersionString) < 9.0) {
+                project.SetBuildProperty(project.GetAllTargetGuids(), ExtendedPBXProject.DeploymentTarget, "9.0");
+            }
+            #elif UNITY_5_4_2_OR_NEWER
+            if (UnityEditor.PlayerSettings.iOS.targetOSVersion < iOSTargetOSVersion.iOS_9_0) {
+                project.SetBuildProperty(project.GetAllTargetGuids(), ExtendedPBXProject.DeploymentTarget, "9.0");
```

Comment:
I see what you are getting at, `UnityEditor.PlayerSettings.iOS.targetOSVersion` is a enum, so instead I will use a boolean that will denote whether we change to 9.0 or not